cmd/install/assets/hypershift_operator_test.go

```diff
@@ -0,0 +1,143 @@
+package assets
+
+import (
+	"fmt"
+	"github.com/onsi/gomega"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestHyperShiftOperatorDeployment_Build_OIDC(t *testing.T) {
```

Comment:
TestHyperShiftOperatorDeploymentBuild?

Reply:
I like the _Build format for specifying the function being tested in the struct